### ISSUE TITLE
net_demo_installer script fails with -r option

### ISSUE THREAD

**Issue by gaurav-dalvi** [contiv/demo#126]
```
TASK [cleanup iptables for vxlan vtep port] ************************************
failed: [node1] (item=4789) => {"changed": true, "cmd": "iptables -D INPUT -p udp --dport 4789 -j ACCEPT -m comment --comment \"contiv network traffic (4789)\"", "delta": "0:00:00.005091", "end": "2016-12-14 13:09:48.309327", "failed": true, "item": 4789, "rc": 1, "start": "2016-12-14 13:09:48.304236", "stderr": "iptables: Bad rule (does a matching rule exist in that chain?).", "stdout": "", "stdout_lines": [], "warnings": []}
failed: [node2] (item=4789) => {"changed": true, "cmd": "iptables -D INPUT -p udp --dport 4789 -j ACCEPT -m comment --comment \"contiv network traffic (4789)\"", "delta": "0:00:00.004972", "end": "2016-12-14 13:09:48.315928", "failed": true, "item": 4789, "rc": 1, "start": "2016-12-14 13:09:48.310956", "stderr": "iptables: Bad rule (does a matching rule exist in that chain?).", "stdout": "", "stdout_lines": [], "warnings": []}
```

**Comment by dvavili** [comment]
Fixed by contiv/ansible#290

**Comment by dvavili** [comment]
Fix verified. Closing this.
